src/formatting_stack/reporters/pretty_printer.clj

```diff
@@ -26,7 +26,7 @@
                          suffix)))))
          (string/join "\n"))))
 
-(defn print-summary [{:keys [summary?]} reports]
+(defn print-summary [{:keys [summary? colorize?]} reports]
```

Comment:
* `^boolean?`
* Add coverage similar to that of `#'unit.formatting-stack.reporters.pretty-printer/print-warnings`?